config/providers/safespring/common-config.yaml

```
@@ -28,10 +28,11 @@ ingressNginx:
   controller:
     config:
       useProxyProtocol: false
-    useHostPort: true
+    useHostPort: false
     service:
-      enabled: false
+      enabled: true
       allocateLoadBalancerNodePorts: true
```

I think you need to set service.annotations, thats why the unit tests are failing
```suggestion
      allocateLoadBalancerNodePorts: true
      annotations: {}
```